src/icons/file/IconFileBase.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+import type { Icon } from '../flowTypes';
+
+type Props = Icon & {
+    children: React.Node,
+    fileClassName: string,
+};
+
+const IconFileBase = ({ children, className = '', fileClassName, height = 32, title, width = 32 }: Props) => {
```

Comment:
Do we want to tie file names and classes together? Or should `fileClassName` change to something like `baseClassName` or `iconClassName`?

Reply:
Updated to `baseClassName`.